src/Proto.Cluster/Cache/CacheInvalidationExtensions.cs

```diff
@@ -32,7 +32,7 @@ public static Props WithPidCacheInvalidation(this Props props)
 
         public static Cluster WithPidCacheInvalidation(this Cluster cluster)
```

Comment:
Can we perhaps refactor so we are able to register cluster plugins against the cluster config?

It seems that might be cleaner, and can be initialized in the correct order on startup.

Reply:
`clusterConfig.WithExtension<TExt>` maybe?

Reply:
Could work. But it would not let you pass config to the extension?